squirrels/_schemas/response_models.py

```
@@ -210,20 +207,71 @@ class CompiledQueryModel(BaseModel):
 ## Project Metadata Response Models
 
-class ProjectVersionModel(BaseModel):
-    major_version: Annotated[int, Field(examples=[1])]
-    data_catalog_path: Annotated[str, Field(examples=["/api/squirrels/v0/project/myproject/v1/data-catalog"])]
-
-class ProjectModel(BaseModel):
-    name: Annotated[str, Field(examples=["myproject"])]
-    version: Annotated[str, Field(examples=["v1"])]
-    label: Annotated[str, Field(examples=["My Project"])]
-    description: Annotated[str, Field(examples=["My project description"])]
+SAMPLE_BASE_URL = f"https://example.com/analytics/project/v1"
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)